mithril-signer/src/message_adapters/from_epoch_settings.rs

```diff
@@ -13,6 +13,8 @@ impl FromMessageAdapter<EpochSettingsMessage, EpochSettings> for FromEpochSettin
             epoch: message.epoch,
             protocol_parameters: message.protocol_parameters,
             next_protocol_parameters: message.next_protocol_parameters,
+            current_signers: SignerMessagePart::try_into_signers(message.current_signers).unwrap(),
+            next_signers: SignerMessagePart::try_into_signers(message.next_signers).unwrap(),
```

Comment:
I guess we don't want to keep these `unwrap` here. 
Maybe the `TryFromMessageAdapter` is more appropriate to implement when the conversion can fail.

Reply:
Done